src/dotgraph.h

```diff
@@ -36,6 +36,7 @@ class DotGraph
                  m_zoomable(TRUE), m_urlOnly(FALSE) {}
     virtual ~DotGraph() = default;
 
+    static void writeGraphHeader(TextStream& t, const QCString& title = QCString());
```

Comment:
Does this need to become public? I would expect keeping it protected would work fine as DotGroupCollaboration publicly inherits from DotGraph.

Reply:
Right. Fixed.

Reply:
@doxygen , would you like to review?